/**:
  ros__parameters:
    costmap_frame: "map"
    vehicle_frame: "base_link"
    map_frame: "map"
    grid_min_value: 0.0
    grid_max_value: 1.0
    grid_resoultion: 0.2
    grid_length_x: 70.0
    grid_length_y: 70.0
    grid_position_x: 0.0
    grid_position_y: 0.0
    use_wayarea: true
    bound_costmap: true
    route_box_padding_m: 10.0
